src/plugin/coreplugins/RestRouterGenerator/Templates/combine_templates.js

```diff
@@ -0,0 +1,84 @@
+/*jshint node:true*/
```

Comment:
why do we use the templates this way?
wouldn't it be more simple to require them as text?